lib/src/main/java/macrobase/analysis/summary/itemset/Apriori.java

```diff
@@ -0,0 +1,141 @@
+package macrobase.analysis.summary.itemset;
+
+import macrobase.analysis.summary.itemset.result.ItemsetWithCount;
+
+import java.util.*;
+
+public class Apriori {
```

Comment:
Oh i ported it over since it's used in the test code for FPGrowth. But I can move this to test.